src/hooks/useSandboxServices.ts

```diff
@@ -45,7 +45,7 @@ export const useSandboxServices = (handleShowAAPModal: () => void): Service[] =>
       // we need to un-idle the instance
       await axiosInstance
         .patch(
-          `/apis/aap.ansible.com/v1alpha1/namespaces/${signupData.defaultUserNamespace}/ansibleautomationplatforms/${data.items[0].metadata.name}`,
+          `/apis/aap.ansible.com/v1alpha1/namespaces/${signupData.defaultUserNamespace}/ansibleautomationplatforms/sandbox-aap`,
```

Comment:
@mfrancisc is this API change documented anywhere? I'm not familiar with this API in particular.

Reply:
this is not actually an API change, it's just that instead of taking the name of the AAP CR from the variable we are safe with using the constant name. We are sure it can be that one since the UI is creating it with this name.